test-lib.sh

```diff
@@ -488,6 +501,13 @@ EOF
     fi
     # Filter out env var definitions from $s2i_args and create Dockerfile ENV commands out of them
     echo "$s2i_args" | grep -o -e '\(-e\|--env\)[[:space:]=]\S*=\S*' | sed -e 's/-e /ENV /' -e 's/--env[ =]/ENV /' >>"$df_name"
+    # Add in artifacts if doing an incremental build
+    if [ $incremental == true ]; then
```

Comment:
Since $incremental might only be true/false, much nicer would be `if $incremental; then` variant.